dapps/organizations/src/i18n/en.ts

```diff
@@ -222,11 +222,11 @@ const i18n: any = {
         },
         "request-error": "Request could not be loaded!",
         "status": {
-          "accepted": "notarially verified",
```

Comment:
Some more translation keys in this file includes `notarial verification`, should be `notary verification`

Reply:
`notarial verification` is OK, I wouldn't change that as long as it isn't a request from PO.